Update tests and examples

davidronk opened the pull request:
* update hosted tests/examples to not use helper method
* update tests to use test api key

Commit:
Update hosted examples and tests to no longer utilize a helper method

Why is this change needed?
--------------------------
The example files were a little confusing about the use case for hosted
documents because they utilized a helper to immediately download a file.

How does it address the issue?
------------------------------
The example files now print out the url where the hosted document can be
publicly downloaded.

While the test files still validate that our methods produce a valid
hosted document, they no longer utilize the get_async_doc method.

Commit:
Change tests to use test api key

Why is this change needed?
--------------------------
It was confusing that the test api key wasn't valid for testing hosted
documents.  It also complicated the tests since they required a valid
paid api key to pass.

How does it address the issue?
------------------------------
DocRaptor has been updated to allow hosted document creation using the
test api key.  This updates the agent to do the same.

AlexOverbeck commented on examples/async_hosted.php:
```diff
@@ -45,9 +45,8 @@
-        $doc_response = $docraptor->getAsyncDoc($status_response->getDownloadId());
         $file = fopen("/tmp/docraptor-hosted-php.pdf", "wb");
-        fwrite($file, $doc_response);
+        fwrite($file, file_get_contents($status_response->getDownloadUrl()));
```
Since DocRaptor is hosting the document for user's we probably don't need to write it to a file any more. It seems good to do this in our test files, but what if our examples just simply echoed the download url?

davidronk replied:
Very good catch 👍

AlexOverbeck reviewed (commented):
This looks good, but see comment below. 👍

Commit:
Only output hosted document url in examples

Why is this change needed?
--------------------------
Since DocRaptor is hosting the final document there's no need to download
the file in the examples.

How does it address the issue?
------------------------------
Only output the document url in the hosted examples.
